lib/modules/datasource/conan/index.ts

```diff
@@ -153,48 +153,58 @@ export class ConanDatasource extends Datasource {
+                .sort((a, b) => semver.sortVersions(a.version, b.version));
 
-            const latestVersion = sortedReleases.at(-1)?.version;
+              const latestVersion = sortedReleases.at(-1)?.version;
 
-            if (!latestVersion) {
-              return dep;
+              if (!latestVersion) {
+                return dep;
+              }
+              logger.debug(
+                `Conan package ${packageName} has latest version ${latestVersion}`
+              );
+
+              const latestRevisionUrl = joinUrlParts(
+                url,
+                `v2/conans/${conanPackage.conanName}/${latestVersion}/${conanPackage.userAndChannel}/latest`
+              );
+              const revResp = await this.http.getJson<ConanRevisionJSON>(
+                latestRevisionUrl
+              );
+              const packageRev = revResp.body.revision;
+
+              const [user, channel] = conanPackage.userAndChannel.split('/');
+              const packageUrl = joinUrlParts(
+                `${groups.host}/artifactory/api/storage/${groups.repo}`,
+                `${user}/${conanPackage.conanName}/${latestVersion}/${channel}/${packageRev}/export/conanfile.py?properties=conan.package.url`
+              );
+              const packageUrlResp = await this.http.getJson<ConanProperties>(
+                packageUrl
+              );
+
+              if (
+                packageUrlResp.body.properties &&
+                'conan.package.url' in packageUrlResp.body.properties
+              ) {
+                const conanPackageUrl =
+                  packageUrlResp.body.properties['conan.package.url'][0];
+                dep.sourceUrl = conanPackageUrl;
+              }
             }
-            logger.debug(
-              `Conan package ${packageName} has latest version ${latestVersion}`
-            );
-
-            const latestRevisionUrl = joinUrlParts(
-              url,
-              `v2/conans/${conanPackage.conanName}/${latestVersion}/${conanPackage.userAndChannel}/latest`
-            );
-            const revResp = await this.http.getJson<ConanRevisionJSON>(
-              latestRevisionUrl
-            );
-            const packageRev = revResp.body.revision;
-
-            const [user, channel] = conanPackage.userAndChannel.split('/');
-            const packageUrl = joinUrlParts(
-              `${groups.host}/artifactory/api/storage/${groups.repo}`,
-              `${user}/${conanPackage.conanName}/${latestVersion}/${channel}/${packageRev}/export/conanfile.py?properties=conan.package.url`
-            );
-            const packageUrlResp = await this.http.getJson<ConanProperties>(
-              packageUrl
-            );
-            const conanPackageUrl =
-              packageUrlResp.body.properties['conan.package.url'][0];
-            dep.sourceUrl = conanPackageUrl;
+          } catch (err) {
+            logger.debug({ err }, "Couldn't determine conan package url");
```

Comment:
```suggestion
            logger.debug({ err }, "Couldn't determine Conan package url");
```

I think we should capitalize the proper noun Conan here?